hedgehog/hedgehog.cabal

```diff
@@ -116,6 +116,9 @@ test-suite test
   hs-source-dirs:
     test
 
+  other-modules:
+      Test.Hedgehog.Text
```

Comment:
Happy to add this, would you be able to align it at the same level as the modules above (so, two space indent):
```
  other-modules:
    Test.Hedgehog.Text
```